src/FsCheck.NUnit/FsCheckPropertyAttribute.fs

```diff
@@ -145,6 +149,12 @@ and FsCheckTestMethod(mi : IMethodInfo, parentSuite : Test) =
         testResult.RecordException(x.FilterException <| ex, failureSite)
 
     member private x.RunTestMethod context testResult =
+        let parseStdGen (str: string) =
```

Comment:
Shamelessly copypasted from xUnit's propery file. Haven't found proper "utils" place.

Reply:
Don't worry about it.